libpod/image/image.go

```diff
@@ -78,6 +78,11 @@ func NewImageRuntimeFromOptions(options storage.StoreOptions) (*Runtime, error)
 	}, nil
 }
 
+// GetStore returns the store from storage
+func (i *Image) GetStore() storage.Store {
```

Comment:
We should not need this. I really do not want to return a Store anywhere the user can get one.

Reply:
If you or @baude have a suggestion around it, I'm happy to do it.  I tried using image.imageruntime.store in the call in runtime_img.go below  but got:

```
 github.com/projectatomic/libpod/libpod
_output/src/github.com/projectatomic/libpod/libpod/runtime_img.go:135:65: image.imageruntime undefined (cannot refer to unexported field or method imageruntime)
```

Thought exposing the function would be better.  However if I move that code to handle the storage containers into image.go as talked about below, that might solve that problem.

Reply:
The image runtime's store is guaranteed to be the same as the normal runtime's store - you should just be able to runtime.store instead

Reply:
Agreed, i went to great pains to avoid this.

Reply:
ack, I was over complicating it, I can just do r.store in runtime_img.go and it gets me what I need.  Thought I'd to pull it from each image and that wasn't resolving.